### PR TITLE
Creates git-workflow reference

### PR DESCRIPTION
**Pull request by Gobleizer**
Please give feedback on the git workflow, will refactor as team sees fit.

PM team please review @ahjohnston 

Thank you

### CODE REVIEW

**Comment by ahjohnston** [comment]
[[1] approve git workflow](https://trello.com/c/VEYH3qCC/26-1-approve-git-workflow)

**Comment by Gobleizer** [comment]
[2 Write up a git Workflow](https://trello.com/c/lJXLykcM/18-2-write-up-a-git-workflow)

File: git-workflow.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```
+origin	https://github.com/PacificDevelopment/JobSite.git (fetch)
+origin	https://github.com/PacificDevelopment/JobSite.git (push)
+```
+
+## Working on a feature
+
+> No work should be done on the `main` branch.
+
+All work should be done on a **feature branch**; that is, an existing or newly created branch designated to the feature on which you are working.
```
Comment (ahjohnston) [inline git-workflow.md:46]: Could we add a line about what to do if a developer accidentally makes a commit to their local main branch?

File: git-workflow.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Resoruce for more about feature branch workflow: https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow
+
```
Comment (ahjohnston) [inline git-workflow.md:49]: Type at "resource"

File: git-workflow.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+git commit -m "<your_commit_message>"
+```
+
+Use descriptive commit messages in the present tense; for example:
+
+```
+git commit -m "implement click event handler on checkout button"
+```
+Resource for writing good commit messages: https://cbea.ms/git-commit/
+
```
Comment (ahjohnston) [inline git-workflow.md:73]: Including outside resources === very helpful!!

File: git-workflow.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    Tech team please review @Gobleizer
+      OR
+    PM please review @ahjohnston
+```
+    You may get any team member to review your work, but a final check and merge must be done by one of the three repo maintainers, AJ, QM, or KG.
+
+Finally:
+
+1. Resolve any merge conflicts.
+2. Add the pull request ID# to the corresponding Trello ticket
```
Comment (ahjohnston) [inline git-workflow.md]: Bonus! Use the Trello/GitHub integration to link the pull request. On the Trello ticket > Power-ups > GitHub> Attach Pull Request > search for/select pull request by name
Screen Shot 2021-12-09 at 12 45 26 AM

**Review by ahjohnston — APPROVED** [review]
Looks wonderful! I left a few comments & optional suggestions

File: git-workflow.md (hunk 2 above)
Comment (Gobleizer) [reply git-workflow.md:49]: Fixed

File: git-workflow.md (hunk 4 above)
Comment (Gobleizer) [reply git-workflow.md]: Thank you, added photo too!

File: git-workflow.md (hunk 1 above)
Comment (Gobleizer) [reply git-workflow.md:46]: Added instructions for what to do for 2 scenarios:
1) I accidentally make commits to main that were intended for an existing feature-branch.
2) I accidentally make commits to main that were intended for a new feature-branch.

File: git-workflow.md (hunk 3 above)
Comment (Gobleizer) [reply git-workflow.md:73]: Happy to help!

**Pull request merged**